server/app/views/dev/DatabaseSeedView.java

```diff
@@ -52,8 +59,37 @@ public Content render(
     String prettyActivePrograms = getPrettyJson(activePrograms);
     String prettyQuestions = getPrettyJson(questionDefinitions);
 
+    ATag indexLinkTag =
+        a().withHref(routes.DevDatabaseSeedController.index().url())
+            .withId("index")
+            .with(submitButton("index", "Go to dev database seeder page"));
+
+    DivTag content =
+        div()
+            .with(h1(title))
+            .with(indexLinkTag)
+            .with(
+                div()
+                    .with(makeCsrfTokenInputTag(request))
```

Comment:
**suggestion (blocking):** remove the CSRF token. Also looks like both of these methods won't need the `request` parameter anymore since they don't need to generate CSRF tokens.

Reply:
CSRF token removed.